website/docusaurus.config.js

```diff
@@ -71,14 +71,14 @@ var siteSettings = {
     },
     announcementBar: {
       id: "biweekly-demos",
-      content: "Join us on May 14th and 15th for the dbt Labs Launch Showcase event.",
+      content: "Join our bi-weekly demos and see dbt Cloud in action!",
```

Comment:
sugg: no hyphen for American English

```suggestion
      content: "Join our biweekly demos and see dbt Cloud in action!",
```

Reply:
great flag @nghi-ly ! i prefer bi-weekly as well. i kept it with the hyphen since that's how our marketing site spells it: https://www.getdbt.com/resources/webinars/dbt-cloud-demos-with-experts but happy with this sugg